cmd/valet_link.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"trellis-cli/trellis"
+)
+
+type ValetLinkCommand struct {
+	UI      cli.Ui
+	Trellis *trellis.Trellis
+}
+
+func (c *ValetLinkCommand) Run(args []string) int {
+	if err := c.Trellis.LoadProject(); err != nil {
+		c.UI.Error(err.Error())
+		return 1
+	}
+
+	var environment string
+
+	switch len(args) {
+	case 0:
+		environment = "development"
```

Comment:
Most commands don't have a default env but I guess this makes sense since Valet should be in development only 👍